core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -930,17 +932,41 @@ private void updateTableMetadata(
   }
 
   static void execute(Connection connection, String sql) throws SQLException {
+    execute(connection, sql, null);
+  }
+
+  static void execute(Connection connection, String sql, @Nullable SQLWarningHandler handler)
+      throws SQLException {
     if (Strings.isNullOrEmpty(sql)) {
       return;
     }
     try (Statement stmt = connection.createStatement()) {
       stmt.execute(sql);
+      throwSQLWarningIfNeeded(handler, stmt);
+    }
+  }
+
+  private static void throwSQLWarningIfNeeded(SQLWarningHandler handler, Statement stmt)
```

Comment:
```suggestion
  private static void throwSqlWarningIfNeeded(SqlWarningHandler handler, Statement stmt)
```

Reply:
Thank you.
f0ce19bac0ac0e1b1df8d21809a4e35fbf90d239